samples/openmp_testsuite/Makefile

```diff
@@ -2,43 +2,38 @@ TOP=$(abspath ../..)
 include $(TOP)/defs.mak
 
 CONTAINER_NAME=omp.myst.apline.062021
-CONTAINER_IMAGE_FLAVOR=alpine
-CONTAINER_IMAGE_VERSION=3.8
-GCC_VERSION=7.5.0
 
-ID=$(shell docker run -d $(CONTAINER_NAME))
+ifdef STRACE
+OPTS = --strace
+endif
 
-all: appdir rootfs
+OPTS += --memory-size=256m
+
+APPDIR=$(CURDIR)/appdir
+
+ROOTFS=$(CURDIR)/rootfs
 
 EXT2FS_SIZE=2147483648
 
-rootfs:
-	sudo $(MYST) mkext2 --force --size $(EXT2FS_SIZE) appdir rootfs
-	sudo truncate --size=-4096 rootfs
+TEST_FILE = passed.tests
 
-appdir:
-	docker build -t $(CONTAINER_NAME) . \
-	--build-arg IMAGE_FLAVOR=$(CONTAINER_IMAGE_FLAVOR) \
-	--build-arg IMAGE_VERSION=$(CONTAINER_IMAGE_VERSION) \
-	--build-arg GCC_VERSION=$(GCC_VERSION)
-	$(MAKE) export
+all: $(APPDIR) $(ROOTFS)
 
-export:
-	docker stop $(ID)
-	docker export $(ID) -o image.tar
-	rm -rf appdir
-	mkdir -p appdir
-	tar xf image.tar -C appdir
+$(ROOTFS):
+	sudo $(MYST) mkext2 --force --size $(EXT2FS_SIZE) $(APPDIR) $(ROOTFS)
+	sudo chown $(USER).$(USER) $(ROOTFS)
```

Comment:
Line 24 is not needed anymore.

mkext2 is unnecessary, we can run the tests as ramfs. Check for `mkcpio` in other samples.